meinberlin/apps/cms/templatetags/menu.py

```diff
@@ -11,3 +11,8 @@ def get_menu(title):
 
     if menu is not None:
         return menu.items.all()
+
+
+@register.simple_tag()
+def get_identifier(item):
+    return id(item)
```

Comment:
tried adding this to it's own block_tag.py file but it wouldn't recognise it and listed the acceptable places to add a template tag, did I miss a step?

Reply:
You just need to add the tag to a file named whatever you ioad in the template and add that file to a folder templatetags and it should be found. Did that not work?